docs/user/security/securing-communications/index.asciidoc

```diff
@@ -166,8 +166,8 @@ If your PKCS#12 file is encrypted, add the decryption password to your <<secure-
 bin/kibana-keystore add elasticsearch.ssl.truststore.password
 --------------------------------------------------------------------------------
 
-TIP: If you used `elasticsearch-certutil` to generate a PKCS#12 file and you did not specify a password, the file is encrypted, and you need
-to set server.ssl.keystore.password to an empty string.
+NOTE: If you used `elasticsearch-certutil` to generate a PKCS#12 file and you did not specify a password, the file is encrypted, and you
+need to set `server.ssl.truststore.password` to an empty string.
```

Comment:
While I was adding the code tags I noticed that `server.ssl.keystore.password` was wrong, it should have been `server.ssl.truststore.password` this whole time 😅